docs/src/_reference/glossary.md

```diff
@@ -7,7 +7,7 @@ weight: 5
 
 {% for item in site.data.glossary %}
 
-<h2>{{ item.term }} <a href="https://docs.meltano.com/reference/glossary#{{ item.shorthand }}">#</a></h2>
+<h2 id="#{{ item.shorthand }}">{{ item.term }}</h2>
```

Comment:
As it is currently, it creates links like https://deploy-preview-7279--meltano.netlify.app/reference/glossary##python-venv (note the double `#`). This occurs in Firefox and Chrome.

Maybe removing the `#` would fix that? I haven't tested this change - it's just a guess.

```suggestion
<h2 id="{{ item.shorthand }}">{{ item.term }}</h2>
```